codegen/codegen.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	defaultDevURL     = "https://releases.rancher.com/kontainer-driver-metadata/dev-v2.11/data.json"
+	defaultDevURL     = "https://releases.rancher.com/kontainer-driver-metadata/dev-v2.11-rke-extended-life/data.json"
```

Comment:
could you update `defaultReleaseURL` as well? it will be used on tag for release

Reply:
is it the same dev branch or release-v2.11-rke-extended-life?